microsoft/testsuites/performance/common.py

```diff
@@ -423,6 +423,27 @@ def perf_ntttcp(  # noqa: C901
                 )
             notifier.notify(ntttcp_message)
             perf_ntttcp_message_list.append(ntttcp_message)
+
+            # Send unified performance messages
+            if udp_mode:
+                client_ntttcp.send_ntttcp_udp_unified_perf_messages(
```

Comment:
Move the methods to be called in create_ntttcp_udp_performance_message and send_ntttcp_tcp_unified_perf_messages